bluepyopt/ephys/models.py

```diff
@@ -31,7 +31,7 @@
 import collections
 import string
 
-from . import create_hoc
+from . import create_hoc, create_acc
```

Comment:
Where does the name 'acc' actually come from? Is it a file extension used by Arbor?

Reply:
The name comes from [Arbor Cable Cell](https://docs.arbor-sim.org/en/latest/concepts/cable_cell.html), described under [Parsable arbor-components and meta-data](https://docs.arbor-sim.org/en/latest/fileformat/cable_cell.html#parsable-arbor-components-and-meta-data). We use the ACC format only for the label-dict and decor here and tie it to the morphology, for which we reuse the original format, in the JSON top-level file. To produce the GUI output shown above, an `arbor.cable_cell` is constructed from the morphology/label-dict/decor and written in its entirety to ACC using `arbor.write_component(<cable-cell>, <filename>)`. 

Regarding the history of the ACC format, maybe @bcumming or other long-term Arbor contributors have further insights.